DiceCoefSteps.txt

```diff
@@ -0,0 +1,43 @@
+Exercise 1
+
+Generating the preprocessing script
+- created timing files using CreateTimingLandscape.py (submitted with this homework)
+- used the loopTiming.sh to subtract one TR from every sampling point for every condition (the script also submitted with this homework)
+- filled in the relevant fields in the uber_subject GUI taking into account the following point:
+	- Change the smoothing kernel size to 6mm FWHM
	- Change the motion threshold to .5 (mm) (adds -regress_censor_motion 0.5 to the afni_proc.py command)
	- Check regress motion derivatives (adds -regress_apply_mot_types demean deriv to the afni_proc.py command)
	- Use the MNI template (adds -tlrc_base MNI_avg152T1+tlrc to the afni_proc.py
command)
	- Uncheck run cluster simulation
	- Setup a single contrast “landmark stimuli with responses vs. only detection stimuli with responses”. The sum of the weights in your contrast vector should be 0.
+	- First TRs to remove per run set to 1
+	- NB! I DID not run 3dREMLfit
+
+-modified the proc.sub_03 script
+	- added -tpattern alt+z to line ~90
+	- added -save_all -AddEdge -deoblique on -master_tlrc 3 to line ~107
+	- added -bout to line ~313
```

Comment:
Ah! This took me awhile to notice, but the problem is that `-bout` changes the index of the contrasts. I ran without `-bout` so my contrast is at index 17. `-bout` adds a lot of other stuff to the stats bucket, so the index has changed (from index 17 to index 63, I think). This means you are computing the overlap for something other than the contrast of interest.  You can rerun the Dice calculations with the stats output you have, but change things like `'stats.sub_03+tlrc[17]'` to `'stats.sub_03+tlrc[T-C_GLT#0_Tstat]'` so that the contrast is correct. Also recall the use of `3dinfo -label` or `3dinfo -label2index` in class today.